draft-terbu-oauth-sd-jwt-vc.md

```diff
@@ -288,26 +288,30 @@ The SD-JWT and the Disclosures would then be serialized by the Issuer into the f
 
 ## Verification and Processing {#vc-sd-jwt-verification-and-processing}
 
-The recipient of the SD-JWT VC MUST process and verify an SD-JWT VC as
-follows:
-
- 1. REQUIRED. Process and verify the SD-JWT as defined in
-Section 6. of [@!I-D.ietf-oauth-selective-disclosure-jwt]. For the
-verification, the `iss` claim in the SD-JWT MAY be used to retrieve the public
-key from the JWT Issuer Metadata configuration (as defined in
-(#jwt-issuer-metadata)) of the SD-JWT VC issuer. A Verifier MAY use alternative
-methods to obtain the public key to verify the signature of the SD-JWT.
+The recipient (Holder or Verifier) of an SD-JWT VC MUST process and verify an
+SD-JWT VC as described in Section 6 of
+[@!I-D.ietf-oauth-selective-disclosure-jwt].
+
+If Key Binding is required, the Verifier MUST verify the Key Binding JWT
+according to Section 6 of [@!I-D.ietf-oauth-selective-disclosure-jwt]. To verify
+the Key Binding JWT, the `cnf` claim of the SD-JWT MUST be used.
```

Comment:
@awoie What is the importance of the last sentence here? Can we give more context?